configure.ac

```diff
@@ -344,6 +344,15 @@ if test ${cross_compiling:-no} = no && ! test -z ${sandbox_shell+x}; then
   else
     AC_MSG_RESULT(disabled)
   fi
+
+  # A dynamically linked shell will not work in the sandbox. 
+  AC_MSG_CHECKING([whether sandbox-shell is statically linked])
+  if ldd $sandbox_shell 2>&1 | grep -q "not a dynamic executable"; then
```

Comment:
Could we skip this check for darwin?
Darwin doesn't come with `ldd`, and we don't have it in our devShell either. Also darwin programs shouldn't link statically to system libraries anyway.

I think something like this should do the job:

```
# system refers to host platform
case $system of
  *-darwin)
    # ldd unsupported on darwin *build* platform, and
    # static linking against system libraries not recommended on darwin,
    # so we don't check that sandbox-shell is statically linked
    ;;
  *)
    <your check>
    ;;
esac
```

Reply:
I kinda forgot about this, sorry :sweat_smile: 

I rebased and applied your suggestion